public/index.html

```diff
@@ -25,9 +25,9 @@
     <nav class="hamburger-nav unclicked">
       <i class="fa fa-bars" aria-hidden="true"></i>
       <ul>
-        <li class="dropdown tab" data-content="aboutme">About Me</li>
-        <li class="dropdown tab" data-content="skills">Skills/Services</li>
-        <li class="dropdown tab" data-content="projects">Projects</li>
+        <li class="dropdown tab" href="/" data-content="/aboutme">About Me</li>
+        <li class="dropdown tab" href="/skills" data-content="skills">Skills/Services</li>
+        <li class="dropdown tab" href="/projects" data-content="projects">Projects</li>
```

Comment:
You're on the right track here, but `<li>` doesn't have an `href` attribute. Once these are proper links, you should be able to remove `navHandler`.